browser/ui/BUILD.gn

```diff
@@ -228,6 +228,14 @@ source_set("ui") {
       "toolbar/brave_bookmark_sub_menu_model.cc",
       "toolbar/brave_bookmark_sub_menu_model.h",
       "toolbar/brave_recent_tabs_sub_menu_model.h",
+      "webui/brave_education/brave_browser_command_handler.cc",
+      "webui/brave_education/brave_browser_command_handler.h",
+      "webui/brave_education/brave_education_ui.cc",
```

Comment:
these should ideally go in their own BUILD.gn unless there would be a circular dependency issue? I think there probably is one for brave_education_ui so it could go directly in webui/brave_education_ui and the rest of the code could go in webui/brave_education with a separate BUILD.gn

Reply:
I took a look at the code in `browser/ui/webui` and none of those folders use their own `BUILD.gn`. Is this a new pattern that we want to establish?

Reply:
it's not a new pattern, there is just a lot of incorrect code